contrib/ftpasswd

```diff
@@ -130,12 +130,12 @@ if ((exists($opts{'passwd'}) && exists($opts{'group'})) ||
 
   # determine to which file to write the passwd entry
   if (defined($opts{'file'})) {
-    $output_file = $opts{'file'}; 
+    $output_file = $opts{'file'};
     print STDOUT "$program: using alternate file: $output_file\n"
 
   } else {
     $output_file = $default_passwd_file;
-  } 
+  }
```

Comment:
I appreciate these whitespace nits, but it does make a PR like this harder to review.  In the future, please separate style changes like this from the actual functional changes.

Reply:
agree+ and sorry about the whitespace issue, not sure why my editor only picked up a few whitespace issue but not all.